pages/sources.md

```diff
@@ -1,35 +1,28 @@
-# Sources
-
-This is where your source analysis of any/all data should go. Delete this introductory material when submitting your assignment.
-
-This source analysis should discuss whatever source(s) you used for this project.
+{% include nav.html %} {% include css.html %}
 
-See the [README.md](../README.md) file for information on how to submit this assignment.
+# Sources
 
-**You SHOULD revise your source analysis if you already did one for the given website.**
----
 
 ## Basic Source Information
 
-* Where is your source located (URL? Or traditional media)
-* What kind of source is it? (Diaries, records of objects, data tables, etc...)
-* Who, or what organization, is responsible for hosting the source (if online), who gathered the information?
+* [https://www.slavevoyages.org/](https://www.slavevoyages.org/)
+* Database on the embarkation and debarkation of Trade routes in the Atlantic
+* National Endowment for the Humanities
```

Comment:
```suggestion
* Funding: [National Endowment for the Humanities](https://www.neh.gov/)
```

Reply:
I feel like the above changes make better use of the bullet list. Otherwise it is confusing to any reader why it just has a url on one line, then a description on another, and then randomly mentions the NEH. In addition, link whenever relevant!